### PR TITLE
presentation

### LINKED ISSUE #116: Add presentation slides

### PR DESCRIPTION
**Pull request by SveaGaenzle**
I added the presentation slides.

Resolves #116

### CODE REVIEW

File: presentation/presentation slides.md
Comment (Raining-Cloud) [inline presentation/presentation slides.md:1]: Would be good if this is named only slides.md, spaces in paths go boom 💥

File: presentation/presentation slides.pdf
Comment (Raining-Cloud) [inline presentation/presentation slides.pdf:1]: same as above :)

**Review by PacoSchatz — APPROVED** [review]
Thanks for adding the slides. And the problem in `pyproject.toml` seems to be nicely solved also.

**Review by Raining-Cloud — CHANGES_REQUESTED** [review]
Some minor changes, rest seems fine 👍

**Review by Raining-Cloud — APPROVED** [review]

**Pull request merged**